desktop-widgets/tab-widgets/TabDiveInformation.cpp

```diff
@@ -125,12 +169,27 @@ void TabDiveInformation::updateData()
 	updateWhen();
 	ui->waterTemperatureText->setText(get_temperature_string(current_dive->watertemp, true));
 	ui->airTemperatureText->setText(get_temperature_string(current_dive->airtemp, true));
-	updateSalinity();
-
-	ui->atmPressType->setEditable(true);
+	if (current_dive->salinity)
+		ui->salinityText->setText(QString("%1g/ℓ").arg(current_dive->salinity / 10.0));
+	else
+		ui->salinityText->clear();
```

Comment:
Why is updateSalinity() removed, and a version inlined here?

Reply:
Fixed, thank you.

Reply:
No, its not. The copied version is till there.

Reply:
My oversight. duplication removed now.